interpreter/src/functions.rs

```diff
@@ -168,6 +168,41 @@ pub fn double(ftx: &FunctionContext, This(this): This<Value>) -> Result<Value> {
     })
 }
 
+// Performs a type conversion on the target.
+pub fn uint(ftx: &FunctionContext, This(this): This<Value>) -> Result<Value> {
+    Ok(match this {
+        Value::String(v) => v.parse::<u64>().map(Value::UInt).unwrap(),
+        Value::Float(v) => {
+            if v > u64::MAX as f64 || v < u64::MIN as f64 {
+                return Err(ftx.error("unsigned integer overflow"));
+            }
```

Comment:
Curious, can a float64 ever be larger than a u64?

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=6e2e3fb86d9d774fc9cd838d18e2ac77

I was almost positive f64 could be larger, but had no concept of how mind-bogglingly large of numbers an f64 can represent if you don't mind the precision being really bad

Reply:
Yep, I clearly misunderstood how f64s were represented under the hood. That is incredible.